geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml

```diff
@@ -37,7 +37,7 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-jupyter_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-jupyter-leaflet_${scala.binary.version}</artifactId>
             <scope>runtime</scope>
```

Comment:
Does this dependency need to be named?

Also, I *think* the original geomesa-jupyter module was ok to ship since it was just templating things...